lib/rubygems/package.rb

```diff
@@ -290,7 +290,7 @@ def add_metadata(tar) # :nodoc:
   ##
   # Builds this package based on the specification set by #spec=
 
-  def build(skip_validation = false, strict_validation = false)
+  def build(skip_validation = false, strict_validation = false, silent: false)
```

Comment:
I also end up avoiding switching from positional to kwargs, just to keep signatures consistent, but in retrospective, we should start to move on at some point 👍.